index.js

```diff
@@ -60,9 +61,9 @@ class Provider {
 
     debug (`parsed: ${JSON.stringify(parsed)}`)
 
-    const { args, defaults } = this.config
-    Object.keys(args).map(k => {
-      if (!args || !args[k]) {
+    const { argTypes, defaults } = this.config
```

Comment:
Alright we have to come to an agreement on this. Or we have an object attached to the config called `args` or `argTypes`. `argTypes` breaks backward compatibility, so it will required every implementing module needs to be updated. `args` doesn't break the backward compatibility, but I also feel like the variable name has been used too often where I sometimes got confused.